examples/passport-login/src/authentication-strategies/twitter.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-passport-login
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {asAuthStrategy, AuthenticationStrategy} from '@loopback/authentication';
+import {StrategyAdapter} from '@loopback/authentication-passport';
+import {bind, extensionFor, inject} from '@loopback/core';
+import {RedirectRoute, Request} from '@loopback/rest';
+import {UserProfile} from '@loopback/security';
+import {Strategy} from 'passport-twitter';
+import {User} from '../models';
+import {mapProfile, PassportAuthenticationBindings} from './types';
+
+@bind(
+  asAuthStrategy,
+  extensionFor(PassportAuthenticationBindings.OAUTH2_STRATEGY),
+)
+export class TwitterOauthAuthorization implements AuthenticationStrategy {
```

Comment:
`TwitterOauthAuthorization` -> `TwitterOauthAuthentication`?

Reply:
Will rename the Google and Facebook ones too.